lib/plugins/aws/package/lib/packageService.js

```diff
@@ -54,6 +57,8 @@ module.exports = {
     const include = this.getIncludes();
     const zipFileName = this.provider.naming.getServiceArtifactName();
 
+    _.set(this.serverless.service, 'package.artifact', zipFileName);
```

Comment:
.... better: saveState should persist only the filename part of the artifact too. Then extendedValidate does not need to strip anything